Userland/Libraries/LibJS/AST.cpp

```diff
@@ -1159,6 +1299,13 @@ ThrowCompletionOr<Reference> MemberExpression::to_reference(Interpreter& interpr
     return Reference { base_value, move(property_name), {}, strict };
 }
 
+// 13.5.1.2 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-delete-operator-runtime-semantics-evaluation
+// 13.5.2.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-void-operator-runtime-semantics-evaluation
+// 13.5.3.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-typeof-operator-runtime-semantics-evaluation
+// 13.5.4.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-unary-plus-operator-runtime-semantics-evaluation
+// 13.5.5.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-unary-minus-operator-runtime-semantics-evaluation
+// 13.5.6.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-bitwise-not-operator-runtime-semantics-evaluation
+// 13.5.7.1 Runtime Semantics: Evaluation, https://tc39.es/ecma262/#sec-logical-not-operator-runtime-semantics-evaluation
```

Comment:
Since we handle `#private in obj` here as well should the link be added here? Or maybe just before the if statement which takes the special case?

https://tc39.es/ecma262/#sec-relational-operators-runtime-semantics-evaluation

Reply:
Commented at wrong position? `#private in obj` is handled in `BinaryExpression::execute()` (line 1082), which already has that link :^)

Reply:
I got fooled by the diffs, sorry.